Common/Orders/Fees/AlphaStreamsFeeModel.cs

```diff
@@ -0,0 +1,163 @@
+                    var fee = Math.Abs(_forexCommissionRate * totalOrderValue);
+                    feeResult = Math.Max(0, fee);
+                    // Forex fees are all in USD
+                    feeCurrency = Currencies.USD;
+                    break;
+
+                case SecurityType.Option:
+                    CashAmount optionsFeeRatePerContract;
+                    if (!_optionFee.TryGetValue(market, out optionsFeeRatePerContract))
+                    {
+                        throw new Exception($"AlphaStreamsFeeModel(): unexpected future Market {market}");
+                    }
+                    feeResult = order.AbsoluteQuantity * optionsFeeRatePerContract.Amount;
+                    feeCurrency = optionsFeeRatePerContract.Currency;
+                    break;
+
+                case SecurityType.Future:
+                    if (market == Market.Globex || market == Market.NYMEX
+                        || market == Market.CBOT || market == Market.ICE
+                        || market == Market.CBOE || market == Market.NSE)
+                    {
+                        // just in case...
+                        market = Market.USA;
+                    }
+
+                    CashAmount futuresFeeRatePerContract;
+                    if (!_futureFee.TryGetValue(market, out futuresFeeRatePerContract))
+                    {
+                        throw new Exception($"AlphaStreamsFeeModel(): unexpected future Market {market}");
+                    }
+                    feeResult = order.AbsoluteQuantity * futuresFeeRatePerContract.Amount;
+                    feeCurrency = futuresFeeRatePerContract.Currency;
+                    break;
+
+                case SecurityType.Equity:
+                    EquityFee equityFee;
+                    if (!_equityFee.TryGetValue(market, out equityFee))
+                    {
+                        throw new Exception($"AlphaStreamsFeeModel(): unexpected equity Market {market}");
+                    }
+                    
+                    //Per trade notional value fees
+                    var tradeFee = equityFee.FeePerTrade * order.GetValue(security);
+
+                    if (tradeFee < equityFee.MinimumFee)
+                    {
+                        tradeFee = equityFee.MinimumFee;
+                    }
+                    
+                    feeCurrency = equityFee.Currency;
+                    //Always return a positive fee.
+                    feeResult = Math.Abs(tradeFee);
+                    break;
+
+                default:
+                    // unsupported security type
+                    throw new ArgumentException($"Unsupported security type: {security.Type}");
+            }
+
+            return new OrderFee(new CashAmount(
+                feeResult,
+                feeCurrency));
+        }
+
+        /// <summary>
+        /// Helper class to handle IB Equity fees
```

Comment:
If this specific the IB, then perhaps naming it as `IBEquityFee` or `InteractiveBrokersEquityFee` maybe be more descriptive.